dev/com.ibm.websphere.org.eclipse.microprofile.faulttolerance.2.1/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin/
```

Comment:
This .gitignore may not be needed as these paths may be covered at the top level

Reply:
You are correct, it is not needed.